.github/workflows/go_lint.yml

```diff
@@ -13,11 +13,17 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-go@v5
+      # Step 1: Checkout the code from the repository
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+
+      # Step 2: Set up Go environment
+      - name: Set up Go
+        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5
```

Comment:
This hash is for [v5.1.10](https://github.com/actions/setup-go/releases/tag/v5.1.0). Have you considered to use [v5.3.0](https://github.com/actions/setup-go/releases/tag/v5.3.0)? It doesn't seem to have any breaking change.